__tests__/components/Panel-test.tsx

```diff
@@ -110,4 +112,22 @@ describe("Panel", () => {
     spyToggle.restore();
   });
 
+  it("should optionally accept a callback", () => {
+    let spyToggle = Sinon.spy(wrapper.instance(), "toggle");
+    let spyOverride = Sinon.spy(wrapper.instance(), "overrideEnter");
+    let onEnter = Sinon.stub();
+    let input = <input />;
+    wrapper.setProps({
+      toggle: spyToggle,
+      overrideEnter: spyOverride,
```

Comment:
`overrideEnter` and `toggle` are not actually a props on `Panel` but methods. This doesn't seem like the right way to test this.

Reply:
You're spying them above, so no need to set them as props.